### DISCUSSION TITLE
Enable translation of field values

### DISCUSSION THREAD

**Discussion by JeltevanBoheemen** [UUDigitalHumanitieslab/I-analyzer#1416] [Ideas]
When defining a corpus in a different language to English, it would be beneficial to be able to pass a translation. This should be optional, as not all corpora are required to be available to international users.

A possible implementation is a 'translate' option for extractors (similar to 'transform') that finds the original key in a dictionary and translates it to its corresponding value. Translation dictionaries should in this case be defined in the corpus definition.

**Comment by JeltevanBoheemen** [comment]
Low priority.

**Comment by JeltevanBoheemen** [comment]
Functionality alreasy possible through `transform` option.

**Comment by BeritJanssen** [comment]
While it is possible to translate during index time,  we may still require translation on the front end for the view (currently, the values for the Dutch Newspapers are saved in Dutch, we might translate keyword values for the filters etc.)
